packages/client/tests/functional/composites/object/update.ts

```diff
@@ -505,7 +508,7 @@ setupTestSuite(({ contentProperty }) => {
         }
       `,
       )
-    } else {
+    } else if (getQueryEngineProtocol() === 'graphql') {
```

Comment:
It might be worth switching to snapshots so that we can have both errors. Not a strong opinion, I am also fine with manual work.

Reply:
Since we are going to do validation really soon (if stars align correctly, already in 4.11 but most likely in 4.12) I've decided to skip them for now and unskip/come up with other way of testing it when we'll work on the messages.